src/components/Cart/CartElement/CartElement.tsx

```diff
@@ -22,7 +22,10 @@ export default function CartElement({
 }: Readonly<CartElementProps>) {
   const { productInfo } = product
 
-  const { count } = useCombinedStore()
+  const items = useStoreData(useCombinedStore, (state) => state.itemsIds)
+
+  const productQuantity = items?.find((item) => item.productId === productInfo.id)
+    ?.productQuantity
```

Comment:
Just for my understanding: why `const { count } = useCombinedStore()` wasn't enough? Have you caught another bug?

Reply:
it is necessary to identify the counter otherwise its quantity will be equal to the quantity of goods in the cart )i.e. if I have three different goods in the cart then the counter in each product will show three)

Reply:
Thanks, I see now 👍